e1000d/src/device.rs

```diff
@@ -6,6 +7,9 @@ use syscall::flag::O_NONBLOCK;
 use syscall::io::Dma;
 use syscall::scheme::Scheme;
 
+// TODO: One // line here to describe what this GROUP of constants is
+// TODO: one /// line describing each constant. Just defining the acronym
```

Comment:
this is really the key: just explaining what each group of constants is and then breaking out the acronym would help a lot here. Complex constants could use a sentance to help people understand.